packages/core/src/pages/index.tsx

```diff
@@ -47,13 +49,16 @@ function Page({
         }}
       />
       <SiteLinksSearchBoxJsonLd
+        type='WebSite'
+        name={settings?.seo?.name ?? storeConfig.seo.name}
         url={storeConfig.storeUrl}
         potentialActions={[
           {
             target: `${storeConfig.storeUrl}/s/?q`,
             queryInput: 'search_term_string',
           },
         ]}
+        {...(publisherId && { publisher: { '@id': publisherId } })}
```

Comment:
Did it work when `publisherId` is a falsy value?

Reply:
when the `publisherId` is a falsy value, this attribute won't be included, which I think it's ok since it doesn't look like a required attr.